superset-frontend/src/explore/components/PropertiesModal/PropertiesModal.test.tsx

```diff
@@ -0,0 +1,273 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React from 'react';
+import { Slice } from 'src/types/Chart';
+import { render, screen, waitFor } from 'spec/helpers/testing-library';
+import fetchMock from 'fetch-mock';
+import userEvent from '@testing-library/user-event';
+import PropertiesModal from '.';
+
+const createProps = () => ({
+  slice: ({
```

Comment:
Can you check if you can use some of the mocks available in `spec/fixtures`? I know that we have some slices and charts already defined there. If something is missing we could increment. We can also use these mocks in `fetch` results.

Reply:
I tried to apply the slice but it didn't work, at least not in a "simple" way...